transformations/americanize_britishize_english/transformation.py

```diff
@@ -0,0 +1,132 @@
+import requests
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def britishize_americanize(string, custom_vocab):
+    """
+    Gets the british to american english dictionary
+    Gets the american to British english dictionary
+    Merges both these dictionaries with the custom vocab dictionary.
+    Parameters:
+        string(str): original string
+        custom_vocab(dict): custom vocab dictionary containing words that are not present in the imported dictionaries
+    Returns:
+        str: String after replacing the words
+    """
+
+    url = "https://raw.githubusercontent.com/hyperreality/American-British-English-Translator/master/data/american_spellings.json"
+    american_british_dict = requests.get(url).json()
+
+    url = "https://raw.githubusercontent.com/hyperreality/American-British-English-Translator/master/data/british_spellings.json"
+    british_american_dict = requests.get(url).json()
+
+    final_dict = {**american_british_dict, **british_american_dict, **custom_vocab}
+
+    string = " ".join([final_dict.get(word, word) for word in string.split()])
+    return string
+
+
+class AmericanizeBritishizeEnglish(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
```

Comment:
You can add TaskType.SentimentAnalysis in the tasks

Reply:
Done